pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -248,6 +248,11 @@ type V2RegistriesConf struct {
 	// potentially use all unqualified-search registries
 	ShortNameMode string `toml:"short-name-mode"`
 
+	// AdditionalLayerStoreAuthHelper is a helper binary that receives
+	// registry credentials pass them to Additional Layer Store for
+	// registry authentication.
+	AdditionalLayerStoreAuthHelper string `toml:"additional-layer-store-auth-helper"`
```

Comment:
`docs/containers-registries.conf.5.md` also needs an update — probably after the all of the rest of the design is settled.